docs/persistence/README.md

```diff
@@ -0,0 +1,158 @@
+# Persistent Data
+
+In this section we will discuss how to work with persistent data in your
+applications and the different options available to you.
+
+Persistent data is data that is stored on disk and survives application
+restarts. This is in contrast to ephemeral data which is stored in memory
+and is lost when the application is restarted.
+
+## Responsibilities
+
+The team is reponsible for any data that is stored in the various storage
+options that are available through the platform. You can read more in the
+[Data Responsibilities](./responsibilities.md) section.
+
+## What should I choose?
+
+Sequence of questions to ask yourself when choosing the right storage option.
+Choose wisely.
+
+```mermaid
+graph TD
+  A[I got data!] --> B[Is it structured?]
+  B --> |Yes| C[Is it events?]
+  B --> |No| D[Is it files?]
+  C --> |Yes| Kafka
+  C --> |No| E[Is it analytical?]
+  D --> |Yes| GCS[Cloud Storage]
+  D --> |No| Opensearch[OpenSearch]
+  E --> |Yes| GBQ[BigQuery]
+  E --> |No| GCSQL[Cloud SQL]
+
+  click GBQ "#bigquery"
+  click GCS "#cloud-storage"
+  click GCSQL "#cloud-sql"
+  click Kafka "#kafka"
+  click Opensearch "#opensearch"
+```
+
+## Storage Comparison
+
+Below is a list of the different storage options available to you.
+
+| Name | Type | Availebility |  Environment | Backup |
+| ---- | ---- | ------------ |  ----------- | ------ |
+| [Kafka](#kafka) | Streaming | ✅ | All | No? |
+| [Cloud Storage](#cloud-storage) | Object | ✅ | GCP | Yes* |
+| [Cloud SQL](#cloud-sql) | Relational | ✅ | GCP | Yes |
+| [BigQuery](#bigquery) | Relational | ✅ | GCP | No |
+| [Elasticsearch](#elasticsearch) | Document | ⚠️ | All | No? |
+| [OpenSearch](#opensearch) | Document | ✅ | All | No? |
+| [InfluxDB](#influxdb) | Time Series | ⚠️ | All | No? |
+| [Redis](#redis) | Key Value | ⚠️ | All | No |
+| [ActiveMQ](#activemq) | Message | ⚠️ | All | No? |
+| [MongoDB](#mongodb) | Document | ⚠️ | GCP | No |
```

Comment:
On-prem Mongo vil jeg tippe har backup.

Reply:
Litt usikker på om MongoDB burde stå her.

Ja, vi har en guide på hvordan du kan kjøre MongoDB i clusteret, men det er ikke sikkert det er slik vi vil at de skal gjøre det (IMHO).

Alternativt så kan nogenlunde samme oppskriften brukes på de fleste databaser, så da kan vi utvide listen med MySQL, Cassandra, Oracle, Couchbase osv. med relativt enkle grep :stuck_out_tongue:.

Reply:
Fjerner Redis og MongoDB fra listen siden det er run it yourself.